### PR TITLE
Requesting addition of "sen-sam" to Kubeflow org

### PR DESCRIPTION
**Pull request by sen-sam**

### CODE REVIEW

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -295,6 +295,7 @@ orgs:
         - scottilee
         - ScrapCodes
         - seong7
+        - sen-sam
```
Comment (connor-mccarthy) [inline github-orgs/kubeflow/org.yaml:298]: This should be one name higher to be alphabetized

**Review by connor-mccarthy — CHANGES_REQUESTED** [review]

**Review by chensun — APPROVED** [review]

**Pull request merged**